tests/src/vitest.config.ts

```diff
@@ -0,0 +1,7 @@
+import { defineConfig } from "vitest/config";
+
+export default defineConfig({
+    test: {
+        // setupFiles: ["./testHelpers.ts"]
```

Comment:
Since you import the file manually and have no other config options, this file is unnecessary.